libs/ng-jexia/tsconfig.lib.prod.json

```diff
@@ -0,0 +1,6 @@
+{
+  "extends": "./tsconfig.lib.json",
+  "angularCompilerOptions": {
+    "enableIvy": false
```

Comment:
is there any issue with Ivy enabled?

Reply:
No issues, it's just a recommendation for the libraries to continue use View Engine up to Angular 10:
![image](https://user-images.githubusercontent.com/5176060/82405743-70a20300-9a6d-11ea-8c89-c5e2ee0b3fed.png)